packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -183,7 +183,12 @@ new UserPool(this, 'myuserpool', {
 As shown in the code snippet, there are data types that are available for custom attributes. The 'String' and 'Number'
 data types allow for further constraints on their length and values, respectively.
 
-Custom attributes cannot be marked as required.
```

Comment:
I think it may be misleading, since there is no way to set them required.

In CloudFormation pointing this out makes sense because custom and non-custom attributes share the same properties [AWS::Cognito::UserPool SchemaAttribute](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cognito-userpool-schemaattribute.html)

That said, I think it is fine to restore this line.

Reply:
We should restore this line. It's clarifying to the user why such an option does not exist.

Reply:
Done ✅